app/src/components/Title.tsx

```diff
@@ -0,0 +1,11 @@
+export default function Title({ text }: { text?: string }) {
+  return (
+    <div className="flex items-center">
+      <div className="bg-blue-950 w-10 h-0.5 mr-2" />
+      <h1 className="uppercase font-mono font-bold text-blue-950">
+        {text ? text : "Default title"}
```

Comment:
Consider making the text mandatory, we would not want to miss one and have a "DEFAULT TITLE" somewhere
```suggestion
        {text}
```